src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -164,34 +175,51 @@ public RequestTranscriptionResponse saveTranscriptionRequest(
 
     @Override
     @Transactional
+    @SuppressWarnings("checkstyle:MissingSwitchDefault")
     public UpdateTranscriptionResponse updateTranscription(Integer transcriptionId,
                                                            UpdateTranscription updateTranscription) {
-
-        TranscriptionEntity transcription = transcriptionRepository.findById(transcriptionId)
+        final var userAccountEntity = getUserAccount();
+        final var transcriptionEntity = transcriptionRepository.findById(transcriptionId)
             .orElseThrow(() -> new DartsApiException(TRANSCRIPTION_NOT_FOUND));
 
-        validateUpdateTranscription(transcription, updateTranscription);
+        validateUpdateTranscription(transcriptionEntity, updateTranscription);
 
-        TranscriptionStatusEntity transcriptionStatusEntity = getTranscriptionStatusById(updateTranscription.getTranscriptionStatusId());
-        transcription.setTranscriptionStatus(transcriptionStatusEntity);
+        final var transcriptionStatusEntity = getTranscriptionStatusById(updateTranscription.getTranscriptionStatusId());
+        transcriptionEntity.setTranscriptionStatus(transcriptionStatusEntity);
         TranscriptionWorkflowEntity transcriptionWorkflowEntity = saveTranscriptionWorkflow(
             getUserAccount(),
-            transcription,
+            transcriptionEntity,
             transcriptionStatusEntity,
             updateTranscription.getWorkflowComment()
         );
-        transcription.getTranscriptionWorkflowEntities().add(transcriptionWorkflowEntity);
+        transcriptionEntity.getTranscriptionWorkflowEntities().add(transcriptionWorkflowEntity);
 
         UpdateTranscriptionResponse updateTranscriptionResponse = new UpdateTranscriptionResponse();
         updateTranscriptionResponse.setTranscriptionWorkflowId(transcriptionWorkflowEntity.getId());
 
         TranscriptionStatusEnum newStatusEnum = TranscriptionStatusEnum.fromId(transcriptionStatusEntity.getId());
 
-        if (newStatusEnum == APPROVED) {
-            notifyTranscriptionCompanyForCourthouse(transcription.getCourtCase());
-            notifyRequestor(transcription, TRANSCRIPTION_REQUEST_APPROVED);
-        } else if (newStatusEnum == REJECTED) {
-            notifyRequestor(transcription, TRANSCRIPTION_REQUEST_REJECTED);
+        final var courtCaseEntity = transcriptionEntity.getCourtCase();
+        switch (newStatusEnum) {
```

Comment:
Should this switch be in a separate method?

Reply:
I'm not sure it overly matters?